cli.py

```diff
@@ -32,6 +32,9 @@
               help="Toggle token based user authentication.")
 @click.option("--serverurl", default="http://localhost",
               help="Set server url", type=str)
+
+@click.option("--openapi","-o" ,default="doc.jsonld",type=click.File('r'),help="Location to Open API doc")
```

Comment:
Why would default OpenAPI file be in `doc.jsonld` format?

Reply:
its just a placeholder , i needed default to point to a file , im using it as a flag

Reply:
Vaibhav default should be None or something, not the default JSON-LD format. I understand you've very less time left but you should always maintain the coding standards.

Reply:
i am using click.file hence it has to be a file otherwise its giving error , now i could create another file but it would be hard to explain its existence as just a placeholder .

Reply:
yes, that'll work.